nasa-gibs/worldview
# WV-2503: Reimplementing WV logo on GIFs

## Description

This puts the WV logo back on GIFs that are created from the animation menu. We previously had to rip out the `canvg` package that was responsible for converting the WV logo from an SVG to a PNG because it was using an outdated version of `xmldom`. Instead of forking the repo and hosting an altered version ourselves I opted to extract the function that we are utilizing and place it in a unique file since it does not require `xmldom`. 

This means we do not need to rely on `canvg` as a dependency. I did have to introduce `stackblur` and `rbgcolor` as dependencies since they are required in the specific function I extracted from `canvg`. 

## How To Test

1. `git checkout wv-2503`
2. `npm ci`
3. `npm run watch`
4. Open the animation menu
5. Click on the `create GIF` logo.
6. Click on the `Create GIF` button within the 'Create An Animated GIF' menu.
7. Verify that the WV logo is visible on the GIF that was created.
